### PR TITLE
create-show-exercises

### PR DESCRIPTION
**Pull request by CaDuyDu**

### CODE REVIEW

**Comment by CaDuyDu** [comment]
ready

File: app/controllers/admin/exercises_answer_controller.rb [hunk 1]
```diff
@@ -0,0 +1,62 @@
+class Admin::ExercisesAnswerController < Admin::AdminBaseController
+  before_action :list_permissions, only: %i(new edit show index)
+  before_action :load_exercises_answer, only: %i(edit show update destroy)
```
Comment (huuphuc93) [inline app/controllers/admin/exercises_answer_controller.rb]: dùng except thay cho only

File: app/controllers/admin/exercises_answer_controller.rb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+  def create
+    @exercises_answer = ExercisesAnswer.new exercise_answer_params
+    @exercises_answer.answer_true!
+    if @exercises_answer.save
+      flash[:success] = t "create_success"
+      redirect_to  admin_exercises_path
+    else
+      flash[:danger] = t "update_failed"
+      redirect_to  new_admin_exercise_path
```
Comment (huuphuc93) [inline app/controllers/admin/exercises_answer_controller.rb]: render :new

File: app/controllers/admin/exercises_answer_controller.rb [hunk 3]
```diff
@@ -0,0 +1,62 @@
+class Admin::ExercisesAnswerController < Admin::AdminBaseController
+  before_action :list_permissions, only: %i(new edit show index)
+  before_action :load_exercises_answer, only: %i(edit show update destroy)
+  before_action :select_option_subjects, only: %i(new edit)
+
+  def index
```
Comment (huuphuc93) [inline app/controllers/admin/exercises_answer_controller.rb]:
```suggestion
  def index; end
```

File: app/controllers/admin/exercises_answer_controller.rb [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+  private
+
+  def exercise_answer_params
+    params.require(:exercise).permit :content, :is_current
+  end
+
+  def load_exercises_answer
+    @exercises_answer = Exercise.find_by id: params[:id]
+    return if (@exercises_answer)
```
Comment (huuphuc93) [inline app/controllers/admin/exercises_answer_controller.rb]: xóa ()

File: app/controllers/admin/exercises_controller.rb [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    4.times do
+      @exercises.exercises_answer << ExercisesAnswer.new
+    end
+  end
+
+  def create
+    @exercises = Exercise.new exercise_params
+    if @exercises.save
+      flash[:success] = t "create_success"
+      redirect_to  admin_exercises_path
```
Comment (huuphuc93) [inline app/controllers/admin/exercises_controller.rb]: sau redirect cách 1 spaces thôi em

**Comment by CaDuyDu** [comment]
ready

**Comment by CaDuyDu** [comment]
https://edu-redmine.sun-asterisk.vn/issues/6066

**Comment by CaDuyDu** [comment]
https://edu-redmine.sun-asterisk.vn/issues/6065

**Comment by CaDuyDu** [comment]
ready

**Comment by CaDuyDu** [comment]
ready

**Pull request merged**